File: Atcoder_#344/ruby/a_spoiler.rb
s = gets.chomp

char = s.chars
first = nil
second = nil

s.length.times do |i|
  if char[i] == "|" && first.nil?
    first = i
  elsif char[i] == "|"
    second = i
    break
  end
end

s.slice!(first..second)

puts s

File: Atcoder_#344/javascript/a_spoiler.js
function main(input) {
  let s = input.trim();
  let first = null;
  let second = null;

  for (let i = 0; i < s.length; i++) {
    if (s[i] === "|" && first === null) {
      first = i;
    } else if (s[i] === "|") {
      second = i;
      break;
    }
  }

  s = s.slice(0, first) + s.slice(second + 1);

  console.log(s);
}

main(require("fs").readFileSync("/dev/stdin", "utf8"));
